flink-python/pyflink/datastream/formats/tests/test_csv.py

```diff
@@ -344,6 +350,24 @@ def _check_csv_use_header_results(test, results):
     test.assertEqual(row['number'], 123)
 
 
+def _create_csv_default_null_value_schema_and_lines() -> Tuple[CsvSchema, List[str]]:
+    schema = CsvSchema.builder() \
+        .add_string_column('string') \
+        .add_number_column('number') \
+        .set_null_value('') \
```

Comment:
Looks good. Some testing suggestions:
I would test a test with  .set_null_value(), set the default? If so we should test that 
I also suggest a test specifying that a non default literal for null works.